### DISCUSSION TITLE
Firmware update

### DISCUSSION THREAD

**Discussion by 4IceG** [4IceG/RM520N-GL#4] [General]
I do not recommend installing RM520NGLAAR01A08M4G_01.200.01.200 (2023-07-20), after installation, my modem often lost connection to the Internet and hung up on LTE. I went back to version RM520NGLAAR03A01M4G_01.202.01.202 (2023-05-12) and everything is OK.

**Comment by GektorUA** [comment]
Regards to the versions this:
RM520NGLAAR**01**A08M4G_**01.200.01.200** (2023-07-20)
older than:
RM520NGLAAR**03**A01M4G_**01.202.01.202** (2023-05-12)
I think, there is dates misunderstood.

**Reply by 4IceG** [reply]
@iamromulan what do you answer to that?

**Comment by iamromulan** [comment]
@4IceG @GektorUA 

The dates are correct when looking at the PDF release notes included.

I know Quectel will sometimes make Firmware specific to an application or at a manufacturers request. That may explain the version numbers. I'm not sure what version is for what though. Asking about it in the Quectel forums may yield a better answer.

A08 works fine for me in GoldenOrb/r00ter and also in an RGMII board. 

Use the firmware that works best for you.

**Comment by MickMorley** [comment]
@4IceG, what board are  you using for your RM520N-GL modem?

**Reply by 4IceG** [reply]
@MickMorley Modem works in my router https://github.com/4IceG/ZBT-Z8102AX-M2.
